tests/tests.rs

```diff
@@ -1828,3 +1828,21 @@ fn type_list() {
     // This can change over time, so just make sure we print something.
     assert!(!lines.is_empty());
 }
+
+#[test]
+fn only_capture() {
```

Comment:
I suspect we need more tests than this. In particular, the case of testing the use of a capture index that sometimes doesn't exist, depending on the match.